token-lending/program/src/state.rs

```diff
@@ -525,3 +596,87 @@ fn pack_decimal(decimal: Decimal, dst: &mut [u8; 16]) {
 fn unpack_decimal(src: &[u8; 16]) -> Decimal {
     Decimal::from_scaled_val(u128::from_le_bytes(*src))
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::math::WAD;
+    use proptest::prelude::*;
+
+    proptest! {
+        #[test]
+        fn borrow_fee_calculation(
+            borrow_fee_wad in 0..WAD,
+            host_fee_percentage in 0..100u8,
+            borrow_amount in 0..u64::MAX,
```

Comment:
```suggestion
            borrow_fee_wad in 0..=WAD,
            host_fee_percentage in 0..=100u8,
            borrow_amount in 0..=u64::MAX,
```

Reply:
Very good catch, I should've been doing this with all of the others too